usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -1,126 +1,138 @@
-            logger.info("-----------------------------------")
+        if options["submission_ids"]:
+            submission_ids = options["submission_ids"]
         else:
-            logger.info("No DABS to load")
-
-        # Data modification happens here, if you don't flag '--safe'
-        # The submission loader is atomic, so one of these failing will not affect subsequent submissions
-        if options["safe"]:
-            logger.info("Exiting script before data load occurs in accordance with the --safe flag.")
+            submission_ids = self.get_incremental_submission_ids()
+
+        if submission_ids:
+            msg = f"{len(submission_ids):,} submissions will be created or updated"
+            if len(submission_ids) <= 1000:
+                logger.info(f"The following {msg}: {submission_ids}")
+            else:
+                logger.info(f"{msg}.")
+            if options["list_ids_only"]:
+                logger.info("Exiting script before data load occurs in accordance with the --list-ids-only flag.")
+                return
+        else:
+            logger.info("There are no new or updated submissions to load.")
             return
 
-        for submission in missing_submissions:
-            submission_id = submission[0]
+        failed_submissions = []
+        for submission_id in submission_ids:
             try:
                 call_command("load_submission", submission_id)
             except SystemExit:
-                logger.info("Submission failed to load: {}".format(submission_id))
-                failed_submissions.append(str(submission_id))
+                logger.info(f"Submission failed to load: {submission_id}")
+                failed_submissions.append(submission_id)
             except Exception:
-                logger.exception("Submission {} failed to load".format(submission_id))
-                failed_submissions.append(str(submission_id))
+                logger.exception(f"Submission {submission_id} failed to load")
+                failed_submissions.append(submission_id)
 
         if failed_submissions:
             logger.error(
-                "Script completed with the following submissions failures: {}".format(", ".join(failed_submissions))
+                f"Script completed with the following {len(failed_submissions):,} "
+                f"submission failures: {failed_submissions}"
             )
             raise SystemExit(3)
         else:
-            logger.info("Script completed with no failures")
+            logger.info("Script completed with no failures.")
+
+    @staticmethod
+    def calculate_since_datetime():
+        since = SubmissionAttributes.objects.all().aggregate(Max("update_date"))["update_date__max"]
+        if since is None:
+            logger.info("No records found in submission_attributes.  Performing a full load.")
+        else:
+            # In order to prevent skips, we're just always going to look back 7 days.  Since submission is a
```

Comment:
```suggestion
            # In order to prevent skips, we're just always going to look back 14 days.  Since submission is a
```

Reply:
Yeah.  I already fixed this is my follow-up branch.  Mind if I just push it then?

Reply:
I'll allow it